erpnext/stock/doctype/item/item.json

```diff
@@ -1120,6 +1120,15 @@
   {
    "read": 1,
    "role": "Manufacturing User"
+  },
+  {
+   "email": 1,
+   "export": 1,
+   "print": 1,
+   "report": 1,
+   "role": "All",
```

Comment:
why is **All** required here ? The issue occurs on a user with **Supplier** role right ? How is that handled ?

Reply:
I think we can take call on that to explain. Actually, this was paired with Rohit, and he just told me to give select permission to all. it will be fixed.